apis/proto/manager/replication/controller/replication_manager.proto

```diff
@@ -27,7 +27,7 @@ import "payload.proto";
 import "google/api/annotations.proto";
 import "pb/gql.proto";
 
-service Replication {
+service ReplicationController {
```

Comment:
without this change, the name is conflicted with agent/replication_manager.proto